filestore/modules/pingrs/README.md

```diff
@@ -0,0 +1,22 @@
+# pingrs
+
+WASM unknown module written in Rust
+
+## How to install
+
+- Install Rust.
+
+- Install the `wasm32-unknown-unknown` target.
+
+```bash
+rustup target add wasm32-unknown-unknown
+```
+
+## How to build module
+
+```bash
+cd modules/pingrs
+cargo build --target wasm32-unknown-unknown --release
+cd ../..
+cp ./modules/pingrs/target/wasm32-unknown-unknown/release/pingrs.wasm ./modules/
+```
```

Comment:
It would be nice to have a Makefile, or better yet [Taskfile](https://taskfile.dev/) for all this.

Reply:
I am thinking to add some support for building rust/zig with these settings to the `mecha` command.